internal/clients/imagebuilder/image_builder_client.go

```diff
@@ -0,0 +1,84 @@
+package imagebuilder
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func RequestImageBuilder(client *http.Client, endpoint string, method string, orgId string, body *strings.Reader) ([]byte, error) {
+	var req *http.Request
+	var err error
+
+	req, err = http.NewRequest(method, endpoint, body)
+	if err != nil {
+		log.Println("Error : something terrible happen -> ", err)
+		return nil, err
+
+	}
+
+	req.SetBasicAuth(os.Getenv("RED_HAT_USER"), os.Getenv("RED_HAT_PASS"))
```

Comment:
You can now use the config structure, is this how authentication will be done in production tho?

Reply:
No, this is probably not correct, I'm testing it out in ephemeral and would love to get something similar for local, but we will get this info in a header and we should pass that onto an image builder, so image builder has the same info as we do.
I'll work on stubbing this header out in local env.

Reply:
I am positive it is not the correct way in general but for the PoC I authorized like that. 
Should I change it?